AsaApi/Core/Public/Ark/ArkApiUtils.h

```diff
@@ -685,50 +811,56 @@ namespace AsaApi
 			return out_actors;
 		}
 
+		FORCEINLINE UMinimapData& GetMinimapData() const
+		{
+			constexpr auto default_path = "Blueprint'/Game/ASA/Minimap/Core/MinimapData_Base.MinimapData_Base'";
+			APrimalWorldSettings& world_settings = *static_cast<APrimalWorldSettings*>(GetWorld()->GetWorldSettings(false, true));
+
+			if(world_settings.CurrentMinimapDataField().uClass)
+				return *static_cast<UMinimapData*>(world_settings.CurrentMinimapDataField().uClass->GetDefaultObject(true));
+
+			return *static_cast<UMinimapData*>(UVictoryCore::BPLoadClass(default_path)->GetDefaultObject(true));
+		}
+
+		FORCEINLINE FMapData& GetMapDataFromMiniMap(UMinimapData& minimap_data, const FVector& reference_point = FVector::ZeroVector) const
+		{
+			if (minimap_data.MinimapDataField().Num() == 1)
+				return *minimap_data.MinimapDataField().GetData();
+
+			for (auto& data : minimap_data.MinimapDataField())
+			{
+				if (PointIntersectsRectangle(reference_point, data.PlayableMinField(), data.PlayableMaxField()))
+					return data;
+			}
+
+			throw std::invalid_argument("Unable To Find MapData.");
+		}
+
 		/**
 		* \brief Converts FVector into coords that are displayed when you view the ingame map
 		*/
-		FORCEINLINE MapCoords FVectorToCoords(FVector actor_position)
+		FORCEINLINE MapCoords FVectorToCoords(const FVector& position) const
```

Comment:
This is a purposeful breaking change to avoid copies and better reflect the intent of the function.

Recompiling plugins against the new api version would resolve the issue -- we can also force export this with the old name with a linker comment.